src/index.js

```diff
@@ -1,15 +1,20 @@
 //@flow
+import logniWrapper from './logging/logniWrapper'
 
 export default class CommonWidget {
-  constructor (rootElement, htmlTemplate) {
-    this.rootElement = this.getRootElement(rootElement)
+  constructor (rootElementId, htmlTemplate) {
+    this.rootElement = this.getRootElement(rootElementId)
     this.htmlTemplate = htmlTemplate
-    this.setHtmlTemplate()
+    this.setHtmlTemplate(rootElementId)
   }
 
-  setHtmlTemplate() {
-    if(this.rootElement) {
+  setHtmlTemplate(rootElementId) {
+    if (this.rootElement) {
       this.rootElement.innerHTML = this.htmlTemplate
+      logniWrapper.debug(`Widget mounted on #${rootElementId} element.`, 7)
```

Comment:
Prosim toto logovani do `getRootElement`, protoze toho se to primo tyka. Fce `setHtmlTemplate` uz jen vyuziva root element a pocita s tim, ze inicializace jiz byla resena jinde.